jekyll/_cci2/architecture.adoc

```diff
@@ -1,9 +1,12 @@
 ---
-description: "This document describes the various services that make up a CircleCI Server v2 installation."
-version:
-- Admin
-- Server v2.x
+description: This document describes the various services that make up a CircleCI
+  Server v2 installation.
+tags:
+- platform:
+  - Admin
+  - Server v2.x
 ---
+
```

Comment:
An extra empty line is being added. Not sure if that changes anything, but it will be nice if it was not the case